Main.java

```diff
@@ -0,0 +1,24 @@
+import java.util.Arrays;
+
+public class Main {
+    public static void main(String[] args) {
+        int[] array = {5, 11, 7, 20};
+
+        Arrays.sort(array);
+
+        for (int i = 0; i < array.length; i++) {
```

Comment:
A new method should be created. In the main method we use the method
Your method should be universal. What about an array will have 10 elements?

Reply:
static int task(int[] arr) { 
        int diff = Integer.MAX_VALUE; 
       for (int i = 0; i < arr.length; i++) { 
            for (int j = i + 1; j < arr.length - 1; j++) { 
                if (Math.abs(arr[i] - arr[j]) < diff) { 
                    diff = Math.abs(arr[i] - arr[j]);             
                  } 
            } 
        } 
        return diff;    
}